spec/governance.md

```diff
@@ -0,0 +1,30 @@
+# Governance
+
+Governance controls the protocol. A user must stake tokens in order to have the right to vote on proposals. Find more about staking [here](./staking.md)
+
+Governance controls the risk parameters of the protocol and which pairs a user can trade. The parameters of the protocol include collateral ratio for a loan, available lending pools, leverage and liquidation amounts.
+
+## Parameters
+
+The parameters of the protocol are:
+
+- C-ratio (Collateral Ratio)
+  - The collateral ratio of loans controls how much a user must put in for a loan. At the start it will be set to 175%, as the protocol matures it will be lowered.
+
+- Lending pools
+  - Lending pools represents which pools are available to lend and borrow from. Lending pools are created via governance.
+
+- Leverage
+  - Leverage is the amount of margin a trader can take. The higher leverage the protocol allows the more at risk the protocol itself becomes. If a trader takes a large amount of leverage in a illiquid market the protocol will have a harder time liquidating the position if the position does not work in favor of the trader
```

Comment:
Can you tell me how this is different to the collateral ratio?

Reply:
collateral ratio is for a user who is lending funds while leverage is out of the scope of c-ratio. The protocol can decide to offer 5x margin thus in the scenario of trading 1 of x they only need to put up .2 of x. While a borrower would need to put up 1.5 of x

Reply:
Okay so we have a multi purpose lending pool with different borrowing requirements. We will need to make sure that someone can't arbitrate over these different requirements i.e. pretend to do a margin trade when really only wanting to loan currency.

Although I agree principally with having just one lending pool to serve all the various requirements, we might want to consider implementing just for margin trading loans first and we can look to broaden the usage of the lending pool later

Reply:
> Okay so we have a multi purpose lending pool with different borrowing requirements. We will need to make sure that someone can't arbitrate over these different requirements i.e. pretend to do a margin trade when really only wanting to loan currency.

Ill make a diagram for the flow. A margin trader should not be able to take a loan without doing a trade, it should be a single tx to take the loan and make the trade. Then selling of the position would return the loan.